rust/operator-binary/src/controller.rs

```diff
@@ -1161,3 +1083,20 @@ fn liveness_probe(trino: &TrinoCluster) -> Probe {
         ..Probe::default()
     }
 }
+
+fn create_tls_volume(volume_name: &str, tls_secret_class: Option<&TlsSecretClass>) -> Volume {
+    let mut secret_class_name = "tls";
+
+    if let Some(tls) = tls_secret_class {
+        secret_class_name = &tls.secret_class;
+    }
+
+    VolumeBuilder::new(volume_name)
+        .ephemeral(
+            SecretOperatorVolumeSourceBuilder::new(secret_class_name)
+                .with_pod_scope()
+                .with_node_scope()
+                .build(),
+        )
+        .build()
+}
```

Comment:
Please ignore my comment about the scopes. I think it's fine to hard-code the scopes here
```suggestion
fn create_tls_volume(volume_name: &str, tls_secret_class: Option<&TlsSecretClass>) -> Volume {
    let secret_class_name = tls_secret_class
        .map(|t| t.secret_class.as_str())
        .unwrap_or("tls");

    VolumeBuilder::new(volume_name)
        .ephemeral(
            SecretOperatorVolumeSourceBuilder::new(secret_class_name)
                .with_pod_scope()
                .build(),
        )
        .build()
}
```